common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -720,13 +720,13 @@ def test_video_component_stores_speed_correctly_for_multiple_videos(self):
         self.video.speed = '1.0'
 
         # open video "B"
-        self.course_nav.go_to_sequential('B')
+        self.course_nav.go_to_vertical('Test Vertical-1')
```

Comment:
Keep your comments up to date.  This one, f'rinstance, should be `# open Test Vertical-1`